doc/prop.xml

```diff
@@ -8,6 +8,36 @@
 #############################################################################
 ##
 
+<#GAPDoc Label="IsChainDigraph">
+<ManSection>
+  <Prop Name="IsChainDigraph" Arg="digraph"/>
+  <Returns><K>true</K> or <K>false</K>.</Returns>
+  <Description>
+    <C>IsChainDigraph</C> returns <K>true</K> if the digraph <A>digraph</A> is
+    isomorphic to the chain digraph with the same number of vertices as
+    <A>digraph</A>, and <K>false</K> if it is not; see <Ref
+      Oper="ChainDigraph"/>.<P/>
+
+    A digraph is a <E>chain</E> if and only if it is connected,
+    not a cycle and both the in and out degree of any vertex is at most one.
```

Comment:
So, in the method for `IsChainDigraph` you require `InDegreeSet(digraph) = [0, 1]` and `OutDegreeSet(digraph) = [0, 1]` which is not exactly the same as what you've written in the documentation. Can you correct this please?

Reply:
Thanks for the suggestion! In the special case of `ChainDigraph(1)`, the in and out degree sets are `[0]`, but I think that still makes my documentation statement incorrect, so I will rewrite that to clarify that case :)